src/main/javacc/copy.jjt

```diff
@@ -10,15 +11,15 @@ package com.google.cloud.spanner.pgadapter.parsers.copy;
 
 public class Copy {
 
-  public static void parse(String stmt) throws Exception {
+  public static void parse(String stmt, CopyTreeParser.CopyOptions options) throws Exception {
     java.io.InputStream stmtStream = new java.io.ByteArrayInputStream(stmt.getBytes());
-    Copy t = new Copy(stmtStream);
+    Copy tree = new Copy(stmtStream);
     try {
-      ASTStart n = t.Start();
-      CopyVisitor v = new CopyDumpVisitor();
```

Comment:
is the CopyDumpVisitor still in use somewhere or should it be deleted in this PR?

Reply:
The file was renamed multiple times: CopyDumpVisitor -> CopyTreeVisitor -> CopyTreeParser